src/behavior-considered-undefined.md

```diff
@@ -170,10 +179,43 @@ reading uninitialized memory is permitted are inside `union`s and in "padding"
 (the gaps between the fields of a type).
 
 
+### Unwinding inappropriately
+[unwinding]: #unwinding-inappropriately
+
+Unwinding with the wrong [ABI][abi] is undefined behavior:
+
+* Causing an unwind into Rust code from a foreign function that was called via a
```

Comment:
Shouldn't this all be at `panic.md#unwinding-ub` now?

Reply:
Hm...seems like the result of a mistake while trying to rebase the branch. I'll remove this section.